exonum-java-binding-core/rust/ejb-app/TUTORIAL.md

```diff
@@ -54,39 +47,41 @@ EJB App configuration is pretty similar to configuration of any other Exonum ser
 with a few additional parameters.
 
 #### Generate Template Config
+Use `--ejb-module-name` for fully qualified name of your `ServiceModule`.
```

Comment:
@Maria-Nosyk do we say `use <option flag> for <option>`, e.g. "Use `--ejb-module-name` for fully qualified name of your `ServiceModule`"?